data_transfer/devices/dreem.py

```diff
@@ -73,11 +81,10 @@ def download_metadata(self) -> None:
 
             # Serial may not exist in lookup, e.g., if Dreem send a device replacement.
             if not device_serial:
-                print(f"Unknown Device:\n   {recording}")
-                # Move onto next record: skips logic below to simplify error handling
-                continue
+                log.debug(f"Unknown Device:\n   {recording}")
+                continue  # Skip record
 
-            patient_id = (
+            patient_id = utils.format_id_patient(
```

Comment:
Do we need to know this, and could that not be something that is logged within `format_id_patient` so that it can be shared across device implementations?

In the case of dreem, an error is logged if DeviceID or PatientID is None.